src/components/HTMLEngineProvider/HTMLRenderers/CustomEmojiRenderer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import type {FC} from 'react';
+import {Platform, View} from 'react-native';
+import type {CustomRendererProps, TPhrasing, TText} from 'react-native-render-html';
+import type {SvgProps} from 'react-native-svg';
+import GlobalCreateIcon from '@assets/images/customEmoji/global-create.svg';
+import CustomEmojiWithDefaultPressableAction from '@components/HTMLEngineProvider/CustomEmojiWithDefaultPressableAction';
+import ImageSVG from '@components/ImageSVG';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+
+// eslint-disable-next-line rulesdir/no-inline-named-export
+export const emojiMap: Record<string, FC<SvgProps>> = {
```

Comment:
Let us minimize the lint overrides

```suggestion
const emojiMap: Record<string, FC<SvgProps>> = {
```